src/plots/cartesian/layout_attributes.js

```diff
@@ -279,22 +279,22 @@ module.exports = {
         pattern: {
             valType: 'enumerated',
             // TODO could add '%H:%M:%S'
-            values: ['%w', '%H', ''],
+            values: ['day of week', 'time of day', ''],
             dflt: '',
             role: 'info',
             editType: 'calc',
             description: [
                 'Determines a pattern on the time line that generates breaks.',
-                'If *%w* - Sunday-based weekday as a decimal number [0, 6].',
-                'If *%H* - hour (24-hour clock) as a decimal number [0, 23].',
-                'These are the same directive as in `tickformat`, see',
-                'https://github.com/d3/d3-time-format#locale_format',
+                'If *day of week* - Sunday-based weekday as a decimal number [0, 6].',
+                'If *time of day* - hour (24-hour clock) as a decimal number [0, 23].',
+                '*day of week* and *time of day* are similar to *%w* and *%H* directives',
+                'applied in `tickformat`, see https://github.com/d3/d3-time-format#locale_format',
                 'for more info.',
                 'Examples:',
-                '- { pattern: \'%w\', bounds: [6, 0], operation: \'[]\' }',
+                '- { pattern: \'day of week\', bounds: [6, 0] }',
                 '  breaks from Saturday to Monday (i.e. skips the weekends).',
-                '- { pattern: \'%H\', bounds: [17, 8] }',
-                '  breaks from 5pm to 8am (i.e. skips non-work hours).'
+                '- { pattern: \'time of day\', bounds: [16, 8] }',
```

Comment:
`bounds` should be [18, 8] ?

Reply:
It seems reasonable (to me) to set `operation' `dflt` back to "()" for the `time of day` case.